docs/server/source/production-deployment-template/easy-rsa.rst

```diff
@@ -69,14 +69,17 @@ The comments in the file explain what each of the variables mean.
    echo 'set_var EASYRSA_REQ_OU "IT"' >> vars
    echo 'set_var EASYRSA_REQ_EMAIL "dev@bigchaindb.com"' >> vars
 
+We follow the convention of modifying the OU to ``ROOT-CA``,
+``MongoDB-Instance``, ``BigchainDB-Instance``, ``MongoDB-Mon-Instance`` and
+``MongoDB-Backup-Instance`` while issuing certificates.
```

Comment:
Why? I always thought OU meant the Organizational Unit within the company, such as "marketing" or "IT". Do we need to include information like this (e.g. "MongoDB-Instance") in the certificate? If so, is there another setting/value where we could store that?